cfgov/jinja2/v1/_includes/templates/render_block.html

```diff
@@ -0,0 +1,9 @@
+{% if block.block.meta.classname %}
+    <div class="block {{ block.block.meta.classname }}">
+        {{ render_stream_child(block) }}
+    </div>
+{% else %}
+    <div class="block">
+        {{ render_stream_child(block) }}
+    </div>
```

Comment:
This can all be simplified as
```
<div class="block 
                   {{ block.block.meta.classname  if block.block.meta.classname else '' }}">
    {{ render_stream_child(block) }}
</div>
```